c-dependencies/js-compute-runtime/js-compute-builtins.cpp

```diff
@@ -321,6 +322,13 @@ static const uint32_t class_flags = 0;
     return false; \
   }
 
+#define INIT_ONLY(name) \
+  if (FetchEvent::instance()) { \
```

Comment:
Just an idle thought, but perhaps it would be better (in this PR or later) to have a global "execution mode" enum that indicates whether we know we're wizening currently? This way we affirmatively assert that we're in the right mode rather than assume the absence of a fetch event implies it; I can imagine a distant future where there might be other-than-fetch events that also invoke the module.

Reply:
Good point, yes! I'll file an issue about this.